infra/build/build_status/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright 2021 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-runner
+
+RUN mkdir -p /opt/oss-fuzz/infra/build_status
+COPY ${OSS_FUZZ_ROOT}/infra/build/functions/* /opt/oss-fuzz/infra/build_status/
```

Comment:
Why copy it into a different directory (`build_status`) than the one it is actually from?
This seems like a bad idea.

Reply:
This is for common dependencies (like hte infra folder in cifuzz)